doc/tutorials/enable_s5.rst

```diff
@@ -70,17 +70,17 @@ Trigger the User VM's S5
 
 On the Service VM side, it uses the ``acrnctl`` tool to trigger the User VM's S5 flow:
 ``acrnctl stop user-vm-name``. Then, the Device Model sends a ``shutdown`` command
-to the User VM through a channel. If the User VM receives the command, it will send an "ACK"
+to the User VM through a channel. If the User VM receives the command, it will send an ``ACKED``
 to the Device Model. It is the Service VM's responsibility to check if the User VMs
-shut down successfully or not, and decides when to power off itself.
+shut down successfully or not, and decides when to power itself off.
```

Comment:
Let's not mix the terms "power off" and "shut(ting) down".  I think "shut(ting) down" might be the more appropriate term. 
```suggestion
shut down successfully or not, and decides when to power the Service VM itself off.
```

Reply:
It is the Service VM's responsibility to check...and **to decide**... (not "decides").